cmd/kubeops/internal/handler/handler.go

```diff
@@ -104,7 +107,10 @@ func WithHandlerConfig(storageForDriver agent.StorageForDriver, options Options)
 			cfg := Config{
 				Options:      options,
 				ActionConfig: actionConfig,
-				ChartClient:  chartUtils.NewChartClient(kubeHandler, options.ClustersConfig.KubeappsClusterName, options.KubeappsNamespace, options.UserAgent),
```

Comment:
We did not catch in this review that the chart client being removed here was for the `options.ClustersConfig.KubeappsClusterName` cluster, whereas below when calling `chart.GetAppRepoAndRelatedSecrets` we used `config.Cluster` instead. Fixed in #2377